src/main/scala/scalismo/common/interpolation/ValueInterpolator.scala

```diff
@@ -51,41 +51,44 @@ trait Interpolator[@specialized(Double, Float) A] {
     var mix: A = first
     var n = 1
     for (next <- rest) {
-      mix = blend(mix, next, n.toFloat / (n + 1))
+      mix = blend(mix, next, n / (n + 1))
```

Comment:
With passing Doubles to the convex combination method i agree.

Why should we not stay with the simpe name for the simple interpolator? Calling the image interpolator ImageInterpolator does not lead to name clashes and seems to me to be the more natural solution.

Further would the numerics not be a more obvious package for this class than common?